### PR TITLE
Add getUser() to AuthAdmin

### PR DESCRIPTION
**Pull request by brfrn169**
## Description

This PR adds the `getUser()` method to the `AuthAdmin` interface.

## Related issues and/or PRs

- https://github.com/scalar-labs/scalardb/pull/1318

## Changes made

- Added the `getUser()` method to the `AuthAdmin` interface
- Revised Javadoc

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [x] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

N/A

## Release notes

N/A

### CODE REVIEW

File: core/src/main/java/com/scalar/db/api/AuthAdmin.java [hunk 1]
```diff
@@ -56,7 +61,8 @@ default void dropUser(String username) throws ExecutionException {
    * @param namespaceName the namespace name of the table
    * @param tableName the table name
    * @param privileges the privileges
-   * @throws ExecutionException if the user does not exist or the operation fails
+   * @throws IllegalArgumentException if the user does not exist or the table does not exist
```
Comment (komamitsu) [inline core/src/main/java/com/scalar/db/api/AuthAdmin.java:64]: > if the user does not exist or the table does not exist

What about namespace?

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
Super minor, but as I left a suggestion, the verb in a comment should assume third-person singular subject?

File: core/src/main/java/com/scalar/db/api/AuthAdmin.java [hunk 2]
```diff
@@ -98,13 +106,25 @@ default void revoke(
...
+   * @throws IllegalArgumentException if the user does not exist or the namespace does not exist
+   * @throws ExecutionException if the operation fails
    */
   default void revoke(String username, String namespaceName, Privilege... privileges)
       throws ExecutionException {
     throw new UnsupportedOperationException("Not supported in the community edition");
   }
 
+  /**
+   * Retrieve a {@link User} for the given username.
```
[the remark this answers is not in the document]
Comment (brfrn169) [reply core/src/main/java/com/scalar/db/api/AuthAdmin.java]: That's correct. Fixed in 886e06634c97ddf21a9ceb6f0e24f7d5b155b192. Thank you for pointing it out.

File: core/src/main/java/com/scalar/db/api/AuthAdmin.java (hunk 1 above)
Comment (brfrn169) [reply core/src/main/java/com/scalar/db/api/AuthAdmin.java:64]: For this method, we specify the table along with its namespace. If the namespace does not exist, it means the table doesn't exist either. Thanks.

**Pull request merged**